.gitignore

```diff
@@ -93,6 +93,7 @@ myvenv/
 ENV/
 env.bak/
 venv.bak/
+pyvenv.cfg
```

Comment:
this file got automatically generated when working inside a `venv` which is why i added it to the list here